modules/gitops-release-notes-1-8-2.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-8-2_{context}"]
+= Release notes for {gitops-title} 1.8.2
+
+{gitops-title} 1.8.2 is now available on {product-title} 4.10, 4.11, and 4.12.
+
+[id="fixed-issues-1-8-2_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, when you configured Dex using the `.spec.dex` parameter and tried to log in to the Argo CD UI by using *Log In Via Openshift*, you were not able to log in. This update fixes the issue. 
++
+IMPORTANT: In the future release of Red Hat Openshift GitOps v1.9, configuring Dex using the `spec.dex` parameter in the ArgoCD CR is planned to be deprecated. Consider using the `.spec.sso` parameter instead. See "Enabling or disabling Dex using .spec.sso".  link:https://issues.redhat.com/browse/GITOPS-2761[GITOPS-2761]
```

Comment:
should there be a link to "Enabling or disabling Dex using .spec.sso"?  Is it assumed people would know where to go to look for this?

Reply:
> Side note not related to this PR: Our docs still use spec.dex

@reginapizza Shall I raise a separate JIRA for this to remove such instances? Please let me know and we can handle it separately.

Reply:
> side note not related to this PR: I think it needs to be clearer in our docs on [this page](https://docs.openshift.com/container-platform/4.11/cicd/gitops/configuring-sso-on-argo-cd-using-dex.html#gitops-creating-a-new-client-in-dex_configuring-sso-for-argo-cd-using-dex) that spec.dex is deprecated and users should instead use spec.sso. It doesn't even have the deprecation notice until the "Disabling Dex" section.

 I agree, it has been only mentioned in the [release notes](https://docs.openshift.com/container-platform/4.11/cicd/gitops/gitops-release-notes.html#gitops-release-notes-1-6-0_gitops-release-notes)

Reply:
@varshab1210 Need clarification. Regarding https://issues.redhat.com/browse/GITOPS-2761, the `spec.dex` parameter in the ArgoCD CR:

- Is it already deprecated and is planned to remove in 1.9?
- Or is it planned to be deprecated and removed in 1.9?

Reply:
As per the discussion with @varshab1210 I am changing the RN text a bit - like this: 
**IMPORTANT:** The `spec.dex` parameter in the ArgoCD CR is deprecated.  In the future release of Red Hat Openshift GitOps v1.9, configuring Dex using the `spec.dex` parameter in the ArgoCD CR is planned to be removed. 

> should there be a link to "[Enabling or disabling Dex using .spec.sso](https://docs.openshift.com/container-platform/4.11/cicd/gitops/configuring-sso-on-argo-cd-using-dex.html#gitops-disable-dex-using-spec-sso_configuring-sso-for-argo-cd-using-dex)" or is it assumed people would know where to go to look for this?

@reginapizza We are not adding a link. 


@reginapizza and @varshab1210: I have raised a JIRA - [RHDEVDOCS-5098](https://issues.redhat.com/browse/RHDEVDOCS-5098) to address the removal of dex in 1.9 OCP GitOps docs. PTAL.

Reply:
> side note not related to this PR: I think it needs to be clearer in our docs on [this page](https://docs.openshift.com/container-platform/4.11/cicd/gitops/configuring-sso-on-argo-cd-using-dex.html#gitops-creating-a-new-client-in-dex_configuring-sso-for-argo-cd-using-dex) that spec.dex is deprecated and users should instead use spec.sso. It doesn't even have the deprecation notice until the "Disabling Dex" section.

@reginapizza and @varshab1210: I am putting the admonition for the deprecation notice at the start of this page. You can look at the revised PR.